src/styles/index.scss

```diff
@@ -353,27 +355,97 @@ div[class*='sidebarViewport'] {
     height: rem(4);
     width: calc(100% - 1.25rem) !important;
   }
-  &__inner .navbar__items:first-child a {
-    font-weight: normal;
+  &__inner {
+    .navbar__items:first-child a {
+      font-weight: normal;
+    }
   }
   &__link {
-    &--active,
+    color: var(--core-color-solid-slate-700);
+    border-radius: var(--core-borderRadius-600);
+    padding: rem(1) rem(2);
+
+    @media screen and (max-width: 1024px) {
+      padding: rem(1.3);
+    }
+
     &:hover {
-      color: var(--colors-coral500);
+      color: rgba(0, 0, 0, 72%);
+      background-color: var(--hover-solid-light-100);
       text-decoration: none;
     }
+
+    &--active {
+      color: var(--colors-coral500);
+    }
   }
   &__brand {
-    margin-right: rem(5);
+    margin-right: rem(10);
+
+    @media screen and (max-width: 1024px) {
+      margin-right: rem(4);
+    }
   }
   &__logo {
     width: rem(12);
   }
+  &__item {
+    font-size: rem(1.4);
+  }
+
+  .external-nav-link {
+    &::after {
+      content: '';
+      display: inline-block;
+      position: relative;
+      bottom: 2px;
+      background-image: url('/img/arrow_up_right.svg');
```

Comment:
Can we use quill-icons instead of using svg?

Reply:
I didn't think so. In style file we can't import icon from there. I downloaded this icon from quill to use here.